src/components/ColorView.tsx

```diff
@@ -27,14 +27,41 @@ function ColorView({ className }: Props) {
         />
       </div>
 
+      <div className="flex py-2">
+        <label>
+          <input
+            name="overrideStyles"
+            type="checkbox"
+            checked={overrideStyles}
+            onChange={() => setOverrideStyles(!overrideStyles)}
+            disabled={!hasAvailableColorStyles()}
+          />
+          <span className="ml-2 text-gray-600 text-xs"
+          >Override styles if available</span>
+        </label>
+      </div>
+
+      <div className="flex py-2">
+        <label>
+          <input
+            name="addSpaces"
+            type="checkbox"
+            checked={addSpaces}
+            onChange={() => setAddSpaces(!addSpaces)}
+          />
+          <span className="ml-2 text-gray-600 text-xs"
+          >Add spaces to slashes</span>
+        </label>
+      </div>
+
```

Comment:
The tailwind custom forms is added, we should probably style this a bit fancier like this:
```tsx
<div className="py-2">
  <label className="inline-flex items-center mt-2">
    <input
      name="overrideStyles"
      type="checkbox"
      className="form-checkbox"
      checked={overrideStyles}
      onChange={() => setOverrideStyles(!overrideStyles)}
      disabled={!hasAvailableColorStyles()}
    />
    <span className="ml-2 text-xs text-gray-600 select-none">
      Override styles if available
    </span>
  </label>

  <label className="inline-flex items-center mt-2">
    <input
      name="addSpaces"
      type="checkbox"
      className="form-checkbox"
      checked={addSpaces}
      onChange={() => setAddSpaces(!addSpaces)}
    />
    <span className="ml-2 text-xs text-gray-600 select-none">
      Add spaces to slashes
    </span>
  </label>
</div>
```
We would also need to extend the tailwind config so the style looks a bit better

`tailwind.config.js`
```js
module.exports = {
  theme: {
    customForms: theme => ({
      default: {
        input: {
          borderRadius: theme("borderRadius.default"),
          backgroundColor: theme("colors.white"),
          borderColor: theme("colors.gray.300"),
          "&:focus": {
            borderColor: theme("colors.gray.500")
          }
        },
        checkbox: {
          color: theme("colors.teal.500"),
          "&:focus": {
            borderColor: theme("colors.gray.500"),
            boxShadow: undefined
          }
        }
      }
    }),
    extend: {}
  },
  variants: {},
  plugins: [require("@tailwindcss/custom-forms")]
};
```